src/app/tests/suites/DL_LockUnlock.yaml

```diff
@@ -270,7 +269,70 @@ tests:
       response:
           error: FAILURE
 
-    - label: "Try to unlock the door with invalid PIN for the third time"
+    - label: "Try to unlock the door with valid PIN for the third time"
+      command: "UnlockDoor"
+      timedInteractionTimeoutMs: 10000
+      arguments:
+          values:
+              - name: "PINCode"
+                value: "123456"
+
+    - label: "Verify that lock state attribute value is set to Unlocked"
+      command: "readAttribute"
+      attribute: "LockState"
+      response:
+          value: 2
+
+    - label: "Lock the door back prior to next tests"
+      command: "LockDoor"
+      timedInteractionTimeoutMs: 10000
+      arguments:
+          values:
+              - name: "PINCode"
+                value: "123456"
+
+    - label: "Verify that lock state attribute value is set to Locked"
+      command: "readAttribute"
+      attribute: "LockState"
+      response:
+          value: 1
+
+    # Engage lockout by trying invalid PIN again 3 times after the reset.
+    # The retry count should have reset to 0 to allow us 3 retry attempts again
+
+    - label: "Read the lockout timeout"
+      command: "readAttribute"
+      attribute: "UserCodeTemporaryDisableTime"
+      response:
+          value: 10
+
+    - label:
```

Comment:
How do we know that the lockout counter was reset?  If it didn't get reset, we would enter lockout now, and our next two attempts would end up in failure, which is all this test is testing for, no?

Reply:
Yes thats true. since it was reset it takes 3 retries to engage lockout not 2. Please let me know if there's a better way to do this :)

Reply:
The way to do this is to do two wrong things, then a right thing, then two more wrong things, and then the right thing again and test that the second right thing (after 4 previous wrong things, but with a right thing in the middle) works.

Reply:
makes sense. I think I was trying to go with - try 2 wrong times, then 1 right (wrong count should be 2). will my fix is now 0. Try 2nd set of 3 wrong times. lockout happens. try 1 right time and see failure. but i guess this doesn't confirm if the 2nd or 3rd wrong time in the 2nd set caused it. will fix.